### PR TITLE
Ct-403 accept or reject button in case detail

### PR DESCRIPTION
**Pull request by EdwardAndress**
- We need a way for KILOs to accept or reject cases from within the app (not only from the email notification)
- Added a button on the case detail page which links to assignments#edit
- This is only visible to the assigned KILO
- The edit and accept_or_reject controller actions are also protected
- This required some minor tweaks in other tests

- From here on we need to be careful about the users that get created in our tests

### CODE REVIEW

**Comment by EdwardAndress** [comment]
All done @aliuk2012

File: spec/features/03_respond_to_drafter_assignment/respond_to_assignment_spec.rb [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 feature 'respond to drafter assignment' do
 
+# <<<<<<< HEAD
+# ||||||| parent of 53a631c... Ct-403 accept or reject button in case detail
+#   given(:drafter)         { create(:user, roles: ['drafter']) }
+
+# =======
+
+# >>>>>>> 53a631c... Ct-403 accept or reject button in case detail
```
Comment (aliuk2012) [inline spec/features/03_respond_to_drafter_assignment/respond_to_assignment_spec.rb]: Too many comments ;)

File: spec/features/03_respond_to_drafter_assignment/respond_to_assignment_spec.rb [hunk 2]
```diff
@@ -10,6 +17,8 @@
     )
   end
 
+  #given(:drafter)         { kase.drafter }
+
```
Comment (aliuk2012) [inline spec/features/03_respond_to_drafter_assignment/respond_to_assignment_spec.rb]: More comments ;)

**Comment by EdwardAndress** [comment]
Done, properly.

File: spec/features/mark_as_sent_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+feature 'A drafter can mark a case as having been responded to' do
+  scenario 'from the detail view of a case they are working on' do
+
+    
+
+  end
+end
```
Comment (aliuk2012) [inline spec/features/mark_as_sent_spec.rb]: Should this file be in this PR?

Comment (EdwardAndress) [inline spec/features/mark_as_sent_spec.rb]: No, it certainly should not, the sneaky little..!

**Comment by EdwardAndress** [comment]
I'll merge this once the tests pass - I had to resolve a merge conflict.

**Pull request merged**